definitions/3.0.0/messageObject.json

```diff
@@ -0,0 +1,159 @@
+    },
+    "tags": {
+      "type": "array",
+      "items": {
+        "oneOf": [
+          {
+            "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+          },
+          {
+            "$ref": "http://asyncapi.com/definitions/3.0.0/tag.json"
+          }
+        ]
+      },
+      "uniqueItems": true
+    },
+    "summary": {
+      "type": "string",
+      "description": "A brief summary of the message."
+    },
+    "name": {
+      "type": "string",
+      "description": "Name of the message."
+    },
+    "title": {
+      "type": "string",
+      "description": "A human-friendly title for the message."
+    },
+    "description": {
+      "type": "string",
+      "description": "A longer description of the message. CommonMark is allowed."
+    },
+    "externalDocs": {
+      "oneOf": [
+        {
+          "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+        },
+        {
+          "$ref": "http://asyncapi.com/definitions/3.0.0/externalDocs.json"
+        }
+      ]
+    },
+    "deprecated": {
+      "type": "boolean",
+      "default": false
+    },
+    "examples": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "anyOf": [
+          {
+            "required": [
+              "payload"
+            ]
+          },
+          {
+            "required": [
+              "headers"
+            ]
+          }
+        ],
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "Machine readable name of the message example."
+          },
+          "summary": {
+            "type": "string",
+            "description": "A brief summary of the message example."
+          },
+          "headers": {
+            "type": "object"
+          },
+          "payload": {}
+        }
+      }
+    },
+    "bindings": {
+      "$ref": "http://asyncapi.com/definitions/3.0.0/messageBindingsObject.json"
+    },
+    "traits": {
+      "type": "array",
+      "items": {
+        "oneOf": [
+          {
+            "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+          },
+          {
+            "$ref": "http://asyncapi.com/definitions/3.0.0/messageTrait.json"
+          },
+          {
+            "type": "array",
+            "items": [
+              {
+                "oneOf": [
+                  {
+                    "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+                  },
+                  {
+                    "$ref": "http://asyncapi.com/definitions/3.0.0/messageTrait.json"
+                  }
+                ]
+              },
+              {
+                "type": "object",
+                "additionalItems": true
+              }
+            ]
+          }
+        ]
+      }
+    }
+  },
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://asyncapi.com/definitions/3.0.0/messageObject.json"
+}
```

Comment:
missing EOF

Reply:
It is just a notice from GitHub, incase it's relevant for your setup to have newlines at the end of files, in this case I am pretty sure it's not 🙂